app/models/purl.rb

```diff
@@ -20,6 +20,29 @@ class Purl < ActiveRecord::Base
   }
 
   ##
+  # Return true targets with always values only if the object is not deleted in
+  # purl mount
+  # @return [Array]
+  def computed_true_targets
```

Comment:
Thanks for the comments. I think I'm convinced to just add this into the `true_targets` method. Just simplifying the name, and taking into consideration @peetucket's great comment about how data going forward this shouldn't be a problem. Also conforms with @LynnMcRae 's comment.